src/main/java/net/pms/dlna/LibMediaInfoParser.java

```diff
@@ -788,9 +843,9 @@ public static int getSpecificID(String value) {
 	}
 
 	public static String getSampleFrequency(String value) {
-		/**
-		 * Some tracks show several values, e.g. "48000 / 48000 / 24000" for HE-AAC.
-		 * We store only the first value.
+		/*
+		 * Some tracks show several values, e.g. "48000 / 48000 / 24000" for
+		 * HE-AAC. We store the first value only.
```

Comment:
knowing that any renderer is able to play using the fallback AAC-LC, which is the last one.
That's said most of them are smart to parse it themselt and extract the correct informations feeding their need.